### ISSUE TITLE
Any plans on continuing the development on this @praeclarum?

### ISSUE THREAD

**Issue by claudiosanchez** [praeclarum/Bind#5]
@praeclarum Any plans on continuing the development on this? would this work as-in in a Xamarin.Android (non-forms) implementation?

**Comment by kooldave98** [comment]
Yeah, seems like @praeclarum has been busy, I really like this library, as well as the EasyLayout. Some ideas I am having for extending this `Bind` library is to implement `Commanding` i.e binding behaviours (methods). So on the UI, if a button is clicked, some method on your ViewModel gets executed.

**Comment by claudiosanchez** [comment]
@kooldave98 Are you going to implement commanding?

**Comment by alexrainman** [comment]
Is this abandonware? I will love to see Commanding, Converters and Xamarin.Android support.

**Comment by kooldave98** [comment]
@alexrainman @claudiosanchez I imagined that this works with any .Net project (including Xamarin.Android) out of the box. Have a look at the unit tests and see how it's been used. (Unless i'm missing something here.)

**Comment by alexrainman** [comment]
It doesn't works on Android: System.NullReferenceException

**Comment by alexrainman** [comment]
For commanding you can get Command.cs from Xamarin.Forms open source code and use V.MyCommand.Execute(param); from your delegate (no binding). I was planning to use this and Fody as an alternative for a simple MvvM framework but not luck.

**Comment by alexrainman** [comment]
I found this: http://www.codeproject.com/Articles/1070662/Data-Binding-in-Xamarin-Android

**Comment by ekalchev** [comment]
I loved the idea of this repo but it has many issues that I had to fix and eventually I started my own implementation based on this repo. https://github.com/ekalchev/Property-Bindings
